.github/workflows/supply-chain.yml

```diff
@@ -0,0 +1,32 @@
+name: supply-chain
+
+on:
+  push:
+    branches:
+     - main
+  pull_request:
+    branches:
+     - main
```

Comment:
Why restrict this to `main`? Shouldn't we be vetting our active release branches, too?

Reply:
I intend to enable this on release branches once they have their supply-chain directories set up. (see the top comment) Running `cargo vet` with no configuration would not work, plus I want the configuration on release branches to import audits from the main branch via a raw GitHub URL, so that will have to be done in a follow-up.